src/main/java/JavaLogFileFilter.java

```diff
@@ -0,0 +1,102 @@
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.regex.Pattern;
+
+import processors.StreamProcessor;
+
+public class JavaLogFileFilter {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/joegotgit/java-log-file-filter-tool/file/2552765348/issues/source?bid=3266573&fileBranchId=3290347#l12)